src/main/kotlin/nl/info/client/zgw/ztc/model/RoltypeListGeneriekParameters.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.info.client.zgw.ztc.model
+
+import jakarta.ws.rs.QueryParam
+import net.atos.client.zgw.ztc.model.generated.OmschrijvingGeneriekEnum
+import java.net.URI
+
+class RoltypeListGeneriekParameters(
+    private val zaaktype: URI,
+    private val omschrijvingGeneriek: OmschrijvingGeneriekEnum? = null
```

Comment:
Why is this nullable? What's the point of using this class if the `omschrijvingGeneriek` is null?

Reply:
Made it non-nullable